controls/V-72083.rb

```diff
@@ -1,68 +1,64 @@
-Technology (NIST) 800-53 and related documents. Comments or proposed revisions
-to this document should be sent via email to the following address:
-disa.stig_spt@mail.mil.
-
-Release Date: 2017-03-08
-Version: 1
-Publisher: DISA
-Source: STIG.DOD.MIL
-uri: http://iase.disa.mil
------------------
-=end
-
 control "V-72083" do
-  title "The operating system must off-load audit records onto a different system or
-media from the system being audited."
+  title "The operating system must off-load audit records onto a different
+system or media from the system being audited."
   desc  "
-    Information stored in one location is vulnerable to accidental or incidental
-deletion or alteration.
+    Information stored in one location is vulnerable to accidental or
+incidental deletion or alteration.
 
     Off-loading is a common process in information systems with limited audit
 storage capacity.
-
-    Satisfies: SRG-OS-000342-GPOS-00133, SRG-OS-000479-GPOS-0022.
   "
   impact 0.5
-  tag "severity": "medium"
   tag "gtitle": "SRG-OS-000342-GPOS-00133"
+  tag "satisfies": ["SRG-OS-000342-GPOS-00133", "SRG-OS-000479-GPOS-00224"]
   tag "gid": "V-72083"
   tag "rid": "SV-86707r1_rule"
   tag "stig_id": "RHEL-07-030300"
-  tag "cci": "CCI-001851"
+  tag "cci": ["CCI-001851"]
+  tag "documentable": false
   tag "nist": ["AU-4 (1)", "Rev_4"]
   tag "subsystems": ['audit', 'auditd', 'audisp']
   tag "check": "Verify the operating system off-loads audit records onto a different
 system or media from the system being audited.
 
-To determine the remote server that the records are being sent to, use the following
-command:
+To determine the remote server that the records are being sent to, use the
+following command:
 
 # grep -i remote_server /etc/audisp/audisp-remote.conf
 remote_server = 10.0.21.1
 
-If a remote server is not configured, or the line is commented out, ask the System
-Administrator to indicate how the audit logs are off-loaded to a different system or
-media.
+If a remote server is not configured, or the line is commented out, ask the
+System Administrator to indicate how the audit logs are off-loaded to a
+different system or media.
 
-If there is no evidence that the audit logs are being off-loaded to another system
-or media, this is a finding."
+If there is no evidence that the audit logs are being off-loaded to another
+system or media, this is a finding."
   tag "fix": "Configure the operating system to off-load audit records onto a
 different system or media from the system being audited.
 
 Set the remote server option in \"/etc/audisp/audisp-remote.conf\" with the IP
 address of the log aggregation server."
+  tag "fix_id": "F-78435r1_fix"
 
-  describe parse_config_file('/etc/audisp/audisp-remote.conf') do
-    its('remote_server') { should match(/^\S+$/) }
+  if file('/etc/audisp/audisp-remote.conf').exist?
+    describe parse_config_file('/etc/audisp/audisp-remote.conf') do
+      its('remote_server') { should match %r{^\S+$} }
```

Comment:
This only works if Kerberos is properly working since things need to be encrypted.

I get the requirement, but this is difficult to test without a proper acceptance test for the whole thing and may be superseded if syslog is set up properly (as per the below). I'm going to let this through because there's not much we can do about it at this point but it needs to be evaluated further.

Reply:
@trevor-vaughan I'm not sure what you want changed. It doesn't seem that we changed the core logic from the original except the secondary `match` statement.

Reply:
I agree, we should let it slide for now, but we should mark it as a review for the next release or two to see if we can find a more general solution.